build/configs/imxrt1020-evk/loadable_elf_apps/defconfig

```diff
@@ -396,7 +396,7 @@ CONFIG_ARCH_BOARD_HAVE_FLASH=y
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
 CONFIG_FLASH_PART_SIZE="2048,512,512,1024,1024,1024,,128,128,"
-CONFIG_FLASH_PART_TYPE="kernel,bin,bin,bin,bin,smartfs,ftl,config,"
+CONFIG_FLASH_PART_TYPE="kernel,bin,bin,bin,bin,jSmartFs,ftl,config,"
 CONFIG_FLASH_PART_NAME="kernel,app1,app1,app2,app2,userfs,ftl,config,"
 # CONFIG_ARCH_BOARD_HAVE_SECOND_FLASH is not set
```

Comment:
Similarly, here, just like in the previous comment.
Second flash partition name needs to be updated too.